README.md

```diff
@@ -2,6 +2,70 @@
 
 A project that presents an iRODS Zone as HTTP v1.1.
 
+## Quickstart (Running from Docker Hub Image)
+
+1. Generate a local configuration JSON file.
+
+```
+docker run irods/irods_http_api --dump-config-template > config.json
+```
+
+2. Edit/update the template `config.json` file (point to your iRODS Zone, remove OIDC and TLS sections, etc.).
+
+```
+vim config.json
+```
+
+3. Launch the HTTP API with your customized configuration file to check for success/errors:
+
+```
+docker run --rm --name irods_http_api -v ./config.json:/config.json:ro -p 9000:9000 irods/irods_http_api
```

Comment:
Does this need more `-it` to view output?

Is this and the daemonized form needed? Perhaps we can leave the daemonized form as an exercise for the reader and step 4 becomes ... "In a new terminal, run the following ...".

Reply:
Hmm, could be - I tested this and was able to see the 'broken' output for debugging (wrong host, bad port, etc.).

I initially only did the `-d` but couldn't see anything, so thought it would be helpful for holding/filling the `config.json` correctly.

Reply:
I think preferring the foreground version (i.e. no `-d`) is good for quickstart. It causes the log output to be shown immediately. Running with `-d` is a exercise for the reader IMO. Especially once they start thinking about how to deploy it in a production environment.

When you launched it without `-it`, were you able to shut it down using `CTRL-C`?

Reply:
yes, single `Ctrl-C` exited cleanly.

Reply:
removed daemonized form.

Reply:
Cool cool.

Reply:
resolving.